src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -572,6 +572,9 @@ public bool LooksGeneric(SyntaxNode simpleName)
         public SyntaxNode GetTargetOfMemberBinding(SyntaxNode node)
             => (node as MemberBindingExpressionSyntax).GetParentConditionalAccessExpression()?.Expression;
 
+        public SyntaxNode GetNameOfMemberBindingExpression(SyntaxNode node)
+            => (node as MemberBindingExpressionSyntax).Name;
```

Comment:
```suggestion
        public SyntaxNode GetNameOfMemberBindingExpression(SyntaxNode node)
            => ((MemberBindingExpressionSyntax)node).Name;
```

no point doing an `as` only to immediately dot off of it.  if it's the wrong type it will crash, just with a less helpful exception.

Reply:
This is the pattern used in most of SyntaxFacts methods.

Reply:
I'll apply your suggestion, but if you want to fix this everywhere in SyntaxFacts, we may open an issue and fix that in a separate PR.

Reply:
Broken windows sound not be copied :-)